### PR TITLE
Core pr 5

### PR DESCRIPTION
**Pull request by cableman**
Ensured that entity cache do not cache library users and fixed provider admin menu link.

### CODE REVIEW

File: ding_provider.module [hunk 1]
```diff
@@ -737,3 +737,14 @@ function ding_provider_build_entity_id($id, $agency = -1) {
 
   return $ting_entity_id;
 }
+
+/**
+ * Implements hook_entity_info_alter().
+ *
+ * Disable entity cache for profile2 entities, so provider profiles are not
+ * cached.
```
Comment (kasperg) [inline ding_provider.module]: It would be very helpful if the comment states _why_ provider profiles should not be cached.

**Comment by kasperg** [comment]
Review complete.

A one minor comment which is also worth taking into account generally.

File: ding_provider.module (hunk 1 above)
Comment (cableman) [inline ding_provider.module]: Comment have been added: The profile should not be cached as they contains sensitive information about the library users.

**Comment by kasperg** [comment]
Reviewed and approved. Thanks!

**Pull request merged**